src/components/DayPickerRangeController.jsx

```diff
@@ -499,7 +499,7 @@ export default class DayPickerRangeController extends React.Component {
       },
     });
 
-    onPrevMonthClick();
+    onPrevMonthClick(this.state.currentMonth.clone());
```

Comment:
So `currentMonth` is already available to you as a destructured aspect of state. What you probably want to pass down is the new month anyways, no?

if you expect `this.state.currentMonth` to equal `currentMonth.clone().subtract(1, 'month')` at this point, you're probably going to have a bad time (setting state is not synchronous in the way you are expecting).  Better to set that to a separate variable and then update and also pass back.